src/Magnum/GL/Framebuffer.h

```diff
@@ -256,6 +256,21 @@ class MAGNUM_GL_EXPORT Framebuffer: public AbstractFramebuffer, public AbstractO
                  */
                 static const InvalidationAttachment Stencil;
 
+                #ifndef MAGNUM_TARGET_GLES2
+                /**
+                 * @brief Invalidate both depth and stencil buffer
```

Comment:
Should maybe be "Invalidate combined depth and stencil buffer" as in the `@requires_gles30` comment below?

Reply:
I copypasta'd from here: https://github.com/mosra/magnum/blob/5f54cc4702ac101c6d598933b2179d659b0c83b9/src/Magnum/GL/Framebuffer.h#L207 But I have no objection to either version 🌞

Reply:
@mosra I vote for updating the docs in Framebuffer.h to clarify the distiction between `Depth|Stencil` and `DepthStencil` :)

Reply:
Ah wait, so if I invalidate `DepthStencil` but the framebuffer has separate depth and stencil attachment, it'll cause a GL error? If it does, then it should indeed say "Combined depth and stencil buffer" in both cases, yes. If not, then what's there is fine.

(Heh, it's been almost 3 months since this got opened? Sorry.)

Reply:
Added the extra tests, seems to work 🙆‍♂️